File: src/main/java/com/ort/fw/config/WolfMansionWebMvcConfigurer.java
package com.ort.fw.config;

import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;

import com.ort.fw.SetAccessContextInterceptor;
import com.ort.fw.SetUserInfoInterceptor;

public class WolfMansionWebMvcConfigurer extends WebMvcConfigurerAdapter {
    @Override
    public void addInterceptors(InterceptorRegistry registry) {
        registry.addInterceptor(new SetAccessContextInterceptor()).addPathPatterns("/**").excludePathPatterns("/static/**");
        registry.addInterceptor(new SetUserInfoInterceptor()).addPathPatterns("/**").excludePathPatterns("/static/**");
    }
}

File: backend/src/main/kotlin/com/ort/app/fw/config/WolfMansionWebMvcConfigurer.kt
package com.ort.app.fw.config

import com.ort.app.fw.interceptor.AccessContextInterceptor
import com.ort.app.fw.interceptor.UserInfoInterceptor
import org.springframework.web.servlet.config.annotation.InterceptorRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

class WolfMansionWebMvcConfigurer : WebMvcConfigurer {

    override fun addInterceptors(registry: InterceptorRegistry) {
        registry.addInterceptor(AccessContextInterceptor()).addPathPatterns("/**").excludePathPatterns("/static/**")
        registry.addInterceptor(UserInfoInterceptor()).addPathPatterns("/**").excludePathPatterns("/static/**")
    }
}